Components/LandingPage.jsx

```diff
@@ -0,0 +1,84 @@
+import { View, Button, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import React from 'react';
+import { Video } from 'expo-av';
+
+const LandingPage = ({ navigation }) => {
+  return (
+    <LinearGradient
+      colors={['#6a11cb', '#2575fc']}
+      start={{ x: 0, y: 0 }}
+      end={{ x: 1, y: 1 }}
+      style={styles.container}
+    >
+      <Video
+        source={require('./Homie.mp4')}
+        rate={1.0}
+        isMuted={true}
+        resizeMode="contain"
+        shouldPlay
+        isLooping
+        style={styles.logoVideo}
+      />
+      <Text style={styles.welcomeText}>Welcome to HomieHub</Text>
```

Comment:
The font of the Welcome text could be a bit more unique. I know we didn't agree on a font, so maybe we can agree on one moving forward.